docs/docs/api/api-examples.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 2
+id: api-examples
+---
+
+# Implementation reference
+
+This guide is intended to showcase the practical application of the AVS Node API using authentic examples. Examples used are:
+
+- EigenDA
+
+## EigenDA Example
+
+### GET /eigen/node/spec-version
+
+### Response
+
+```
+{
+    "api_version": "v1.0.0"
+}
+```
+
+### GET /eigen/node/version
+
+### Response
+
+```
+{
+  "version": "EigenDA-AVS/v1.0.0"
+}
+```
+
+### GET /eigen/node/health
+
+### Response
+
+```
+200 OK
+```
+
+### GET /eigen/node/services
+
+### Response
+
+```
+{
+  "services": [
+    {
+      "id": "graph-node-da",
+      "name": "graph-node",
+      "description": "Indexer graph node description",
+      "status": "Up",
+    },
+    {
+      "id": "ipfs-da",
+      "name": "ipfs",
+      "description": "ipfs instance description",
+      "status": "Down",
+    },
+    {
+      "id": "postgres-da",
+      "name": "postgres",
+      "description": "Postgres DB description",
+      "status": "Initializing",
+    }
+  ]
+}
+```
+
+### GET /eigen/node/services/graph-node-da/health
+
+### Response
+
+```
+200 OK
+```
+
+### GET /eigen/node/services/ipfs-da/health
+
+### Response
+
+```
+206 Partial Content
+```
+
+### GET /eigen/node/services/postgres-da/health
+
+### Response
+
+```
+503 Service Unavailable
+```
```

Comment:
Are there the status codes inverted here? If the `ipfs-da` service is `Down` then `503` makes more sense, and if `postgres-da` is `Initializing`, `206` fits better because we stated in the doc that `206` means _It is either initializing or not ready yet_

Reply:
Nicely noticed as well! Done [7bde67a](https://github.com/NethermindEth/el-node-spec/pull/52/commits/7bde67ae3375d64e771e82f0d3cd5b088bb12478)